[2025-03-05 01:32:37 +0100] [11735] [INFO] Starting gunicorn 23.0.0
[2025-03-05 01:32:37 +0100] [11735] [ERROR] Connection in use: ('0.0.0.0', 5000)
[2025-03-05 01:32:37 +0100] [11735] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 98] Address already in use
[2025-03-05 01:32:38 +0100] [11735] [ERROR] Connection in use: ('0.0.0.0', 5000)
[2025-03-05 01:32:38 +0100] [11735] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 98] Address already in use
[2025-03-05 01:32:39 +0100] [11735] [ERROR] Connection in use: ('0.0.0.0', 5000)
[2025-03-05 01:32:39 +0100] [11735] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 98] Address already in use
[2025-03-05 01:32:40 +0100] [11735] [ERROR] Connection in use: ('0.0.0.0', 5000)
[2025-03-05 01:32:40 +0100] [11735] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 98] Address already in use
[2025-03-05 01:32:41 +0100] [11735] [ERROR] Connection in use: ('0.0.0.0', 5000)
[2025-03-05 01:32:41 +0100] [11735] [ERROR] connection to ('0.0.0.0', 5000) failed: [Errno 98] Address already in use
[2025-03-05 01:32:42 +0100] [11735] [ERROR] Can't connect to ('0.0.0.0', 5000)
Running the mlflow server failed. Please see the logs above for details.
